kinetic/distribution.yaml

```diff
@@ -5498,6 +5498,29 @@ repositories:
       url: https://github.com/tork-a/minas.git
       version: master
     status: developed
+  mir_robot:
+    doc:
+      type: git
+      url: https://github.com/dfki-ric/mir_robot.git
+      version: kinetic
+    release:
+      packages:
+      - mir_actions
+      - mir_description
+      - mir_driver
+      - mir_gazebo
+      - mir_msgs
+      - mir_navigation
+      tags:
+        release: release/kinetic/{package}/{version}
+      url: https://github.com/uos-gbp/mir_robot-release.git
+      version: 1.0.0-0
+    source:
+      test_pull_requests: true
```

Comment:
Just checking: Have you followed the [instructions](http://wiki.ros.org/buildfarm/Pull%20request%20testing) to provide access to this repo or the org unit to the @ros-pull-request-builder user? If not, please either remove this line or provide the necessary access as described in the wiki page.